### ISSUE TITLE
Testing with Drupal, mostly working but having some issues.

### ISSUE THREAD

**Issue by brantwynn** [brainblocks/brainblocks#9]
I filed a full report on the project at github.com/brantwynn/brainbocks-drupal   
https://github.com/brantwynn/brainblocks-drupal/issues/1

Basically, I've got the block rendering and accepting my settings, but its not behaving the same on a Drupal-rendered page as it is when testing the example code from brainblocks.io on a flat html page.

(Apologies if this does not meet the contribution guidelines - just trying to get another example of brainblocks working out into the wild.)

**Comment by bluepnume** [comment]
Thanks for the report! I'll try to take a look tonight. Do you have an example drupal page live where I can try it out?

**Comment by brantwynn** [comment]
I can launch something for you to try out.  Are you on the xrb discord or is there a way I can send you some login credentials another way?

FYI, I just tested one more time using just 0.000001 XRB and it did work! 🔥🔥🔥

So maybe there was a timeout or some other issue. 

I was too stupid to save the original hash after I took a partial screenshot (seen in the other issue), but I do have the verified hash from this most recent transaction if you want to check on it.
